compat/Makefile.am

```diff
@@ -25,7 +25,6 @@ libcompatsquid_la_SOURCES = \
 	compat.h \
 	compat_shared.h \
 	cppunit.h \
-	cpu.h \
```

Comment:
Since we initially added it the various mixed OS filenames have been renamed and settled on that strange sched.h name.  That one OS file is the include providing all the CPU_* macros, the sched_* functions and the cpuset types. That is why it should be called compat/sched.* per libcompat source layout design.

Reply:
>  That one OS file is the include providing all the CPU_* macros, the sched_* functions and the cpuset types

That statement is very inaccurate AFAICT: FreeBSD does not even have `<sched.h>`, and there is [<sys/cpuset.h>](https://github.com/freebsd/freebsd-src/blob/master/sys/sys/cpuset.h) that provides the `CPU_AND` macro that we are wrapping (among other things).

Overall, I see no compelling reason to rename this file. I think the current `cpu.h` name is overall better than `sched.h` and cannot contradict any reasonable design rules. However, if you insist, please ask the author to rename again, and I will not re-raise this issue in this PR. I consider my change request addressed.